src/libasr/asr_verify.cpp

```diff
@@ -420,6 +420,11 @@ class VerifyVisitor : public BaseWalkVisitor<VerifyVisitor>
     }
 
     void visit_Function(const Function_t &x) {
+        ASR::FunctionType_t* v_func_type = ASR::down_cast<ASR::FunctionType_t>(x.m_function_signature);
+        if (v_func_type->m_abi == abiType::Interactive) {
+            // This function would have been verified in the previous interactive pass
+            return;
+        }
```

Comment:
Instead of just returning here, let's verify things that we can verify. Like I think for interactive functions we expect the function bodies to be empty, we expect the dependencies to be empty and the other things in that direction.